orc8r/cloud/deploy/terraform/orc8r-helm-aws/yace.tf

```diff
@@ -54,7 +54,7 @@ resource "helm_release" "yace_exporter" {
   count = var.cloudwatch_exporter_enabled ? 1 : 0
 
   name       = "yace-exporter"
-  namespace  = kubernetes_namespace.monitoring.metadata[0].name
+  namespace  = kubernetes_namespace.monitoring[0].metadata[0].name
   repository = "https://mogaal.github.io/helm-charts/"
```

Comment:
There would only be one namespace for monitoring (since we will not create one if orc8r_is_staging_deployment is true). Given that, why would be need a [0] for monitoring snamespace?

Reply:
The problem I found was: when I add count [terraform sees the resource as a list](https://www.terraform.io/docs/language/expressions/references.html#resources) not an object anymore. Then access with dot only fails, and I had to include the bracket notation.

Reply:
Thanks for clarifying @joary